website/docs/how-prysm-works/ethereum-public-api.md

```diff
@@ -0,0 +1,34 @@
+---
+id: ethereum-public-api
+title: Ethereum 2.0 REST API
+sidebar_label: Ethereum 2.0 API 
+description: This section contains information about the official Ethereum 2.0 REST API.
```

Comment:
I decided to stick with the `2.0` nomenclature for now. We should probably swap it out all at once.